README.md

```diff
@@ -91,8 +93,10 @@ The forecast start dates in this dataset are from 2020/01/02 to 2020/12/31.
 The `hindcast-like-observations` (`training-output-reference`) dataset.
 The `forecast-like-observations` (`test-output-reference`) dataset.
 
-The observations are the ground truth to compare with the ML model output and evaluate them. It consists in observation from instruments of temperature and accumulated total precipitation. (TODO add more descriptions) (point to the scripts to create them ? TODO).
-Generally speaking, only past data can be used by the ML models to perform their forecast :
```

Comment:
they can also use the forecasts issued on the same date. see my suggestion below